docassemble/GithubFeedbackForm/github_issue.py

```
@@ -6,6 +6,11 @@
 from docassemble.base.util import log, get_config, interview_url
 import re
 
+try:
+    import google.generativeai as genai
+except:
```

I've been taught to be wary of `except: pass`. Maybe we could specify an error type? For example:

```
except ImportError:
    pass
```

This way import errors are ignored, but it wouldn't hide away other potential errors.